conda_smithy/templates/pixi.toml.tmpl

```diff
@@ -0,0 +1,51 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: yaml -*-
+
+[project]
+name = "{{ feedstock_name }}"
+version = "{{ smithy_version }}"
+description = "Pixi configuration for conda-forge/{{ feedstock_name }}"
+authors = ["@conda-forge/{{ feedstock_name[:-11] }}"]
+channels = ["conda-forge"]
+platforms = {{ platforms }}
+
+[dependencies]
```

Comment:
Do we need to remove the ci setup package from conda in this list if local ci setup is in use?

Reply:
Well actually not since we want the deps.